modules/sitestudio_config_management/sitestudio_config_management.install

```diff
@@ -19,17 +19,20 @@ function sitestudio_config_management_install($is_syncing) {
     $updated_ignore_config = array_unique(array_merge($existing_ignore_config, $new_ignore_config));
     $config->set('ignored_config_entities', $updated_ignore_config);
     $config->save(TRUE);
+    if (!\Drupal::config('cohesion.sync.settings')->get('enabled_entity_types')) {
+      _sitestudio_config_management_import_configuration('cohesion.sync.settings', 'sitestudio_config_management');
```

Comment:
@deepakmishra2 This seems to be incorrect. We can't rewrite whole configuration, as user made changes would be lost with this. For ex: User might have set `package_export_limit` and/or `full_export_limit` to 20, 30 or any other.

Reply:
@vishalkhode1 done.